src/commands/settings/news/channel.ts

```diff
@@ -30,43 +31,35 @@ export default class Channel {
     const guild = interaction.guild;
     const channelOption = interaction.options.getChannel("channel");
     const roleOption = interaction.options.getRole("role");
-
     const channel = guild.channels.cache.get(channelOption?.id);
     const role = guild.roles.cache.get(roleOption?.id ?? "");
 
-    if (role?.name === "@everyone") {
-      return await interaction.followUp({ embeds: [errorEmbed("You can't ping @everyone.")] });
-    }
+    if (role?.name === "@everyone") return await interaction.followUp({
+      embeds: [errorEmbed("You can't ping @everyone.")]
+    });
```

Comment:
```suggestion

```
Though, why? Small servers might really like this feature, and it can't even be abused or something because it can only be used by admins. I kinda disagree with this one I think...

Reply:
ok true we might allow pinging everyone

there may be a time when something urgent happens (0.1a1 for example)

Reply:
next commit has it removed